### PR TITLE
Add `kingdom_taxid` column to `taxon_counts`

### PR DESCRIPTION
**Pull request by tfrcarvalho**
* Add `kingdom_taxid` column to `taxon_counts`
* Will update database manually:
`UPDATE taxon_counts tc
	LEFT JOIN taxon_lineages AS tl ON (
		MOD(ABS(tc.tax_id), 100000000) = tl.taxid
	)
SET tc.kingdom_taxid = tl.kingdom_taxid
WHERE 
	tc.created_at BETWEEN tl.started_at AND tl.ended_at;`

### CODE REVIEW

File: app/models/pipeline_run.rb [hunk 1]
```diff
@@ -368,6 +368,7 @@ def db_load_taxon_counts
     update_names
     # denormalize superkingdom_taxid into taxon_counts
     if multihit?
+      update_kingdoms
```
Comment (cdebourcy) [inline app/models/pipeline_run.rb]: Can we squash update_kingdoms & update_superkingdoms into one?

Comment (tfrcarvalho) [reply app/models/pipeline_run.rb]: done

Comment (cdebourcy) [reply app/models/pipeline_run.rb]: Thanks. I meant put it into a single SQL query though. Can't you just add another column/value pair to the SET clause? Am I missing something?

**Comment by cdebourcy** [comment]
Can this be closed?

**Pull request closed without merging**